<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout d'un nouveau logiciel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Name">
     <property name="title">
      <string>Informations générales:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Nom du logiciel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_editor">
        <property name="text">
         <string>Editeur:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_editor"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_Serial">
     <property name="title">
      <string>Numéro de série:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_SerialReq">
          <property name="text">
           <string>Un numéro de série est-il requis
pour l'installation?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_serialReq">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Non</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Oui</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_Serial">
        <property name="text">
         <string>Si oui, veuillez entrer le numéro de série ci-dessous:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_serialList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_infos">
     <property name="title">
      <string>Informations complémentaires:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_category">
          <property name="text">
           <string>Catégorie du logiciel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_category">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OS (système d'exploitation)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bureautique</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Programmation - Web</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Programmation - Objet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Programmation - Autres</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Education</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gravure</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Audio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Photo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - Action</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - RPG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - MMO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - Stratégie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - Simulation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - FPS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jeux - Autres</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Autres</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_price">
          <property name="text">
           <string>Prix d'achat:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_price">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_euro">
          <property name="text">
           <string>€</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
